engine/src/test/java/io/camunda/zeebe/engine/state/migration/to_8_4/ColumnFamilyPrefixCorrectionMigrationTest.java

```diff
@@ -272,7 +272,7 @@ void shouldMovePiKeyByProcDefKeyToCorrectColumnFamily() {
     @Test
     void shouldIgnoreProcessInstanceKeyByDefinitionKeyEntries() {
       // given
-      decisionRequirementsId.wrapString("decisionRequirements");
```

Comment:
❓ Is this because the length is too large and it would catch it already before (on the length)?

Reply:
If this is the case, could we still have both? So we test all paths?

Reply:
No, it reaches the faulty wrap method here, but it would still allow wrapping and not throw an exception

https://github.com/camunda/zeebe/blob/22af24c71d1f19eda7366b843c46ab9b133f3aac/engine/src/main/java/io/camunda/zeebe/engine/state/migration/to_8_2/corrections/ColumnFamily49Corrector.java#L83-L95

Reply:
@Zelldon Do you think we need another resolution? Or shall I merge?

Reply:
>If this is the case, could we still have both? So we test all paths?

Well, we could do that 👍

Reply:
How 🤔 processInstanceKeyByProcessDefinitionKey is a Long + Long ? So 16 length🤔 decisionRequirements would be 20 chars + the length in front. I'm wondering why this works.

Reply:
Wrapping doesn't is limited by a maximum, but not an exact size. So, if it can fit the first 16 bytes as two longs than it's fine

Reply:
By the way, I tried with the original string, and that passed as well. I don't want to add it now, because it doesn't matter and slows down the release. Let's just merge this.